### ISSUE TITLE
Form Customization shows "spoilers"

### ISSUE THREAD

**Issue by mindeffects** [modxbot/migrate#3960]
mindeffects created Redmine issue ID 3960

Everytime a users edits a (form customized) resource he first sees ALL tabs/fields and only after a second or so the "hidden ones" disappear.

Is there a way to hide these elements in the first place? That would be SO AWESOME because the users won't get the feeling of being somehow "restricted".

**Comment by gerritvanaaken** [comment]
gerritvanaaken submitted:

This is a duplicate of this issue: http://bugs.modx.com/issues/3113

**Comment by mindeffects** [comment]
mindeffects submitted:

Thanks, Gerrit. Love to see, that this is already a thing for 2.1.0-rc1!
